src/js/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Button = ({ title, handleOnClick }) => {
+  const style = {
+    height: "2em",
+    fontSize: "1.5em",
+    borderRadius: "3px",
+    cursor: "pointer"
+  };
+
+  return (
+    <div>
+      <button style={style} onClick={handleOnClick}>
+        {title}
+      </button>
+    </div>
+  );
+};
+
+export default Button;
```

Comment:
@madisoncarr I'm ok with this button for now because I think our styling is going to change dramatically and there's no sense in working too hard on it, but the biggest issue here is that they don't reflow to mobile very well. The text goes outside the box so we'd have to get clever with our CSS to fix that. Nothing to do now, though.

Reply:
@wade00 Sounds good - I figured that our buttons would probably look very different when we're done with it, but the current buttons not responsive at all lol. The way you styled the app appears to be super responsive though - would love to talk more on responsive design if you have some time the next time we meet up.

Reply:
@madisoncarr One thing I would recommend is to use `children` instead of a `title` prop for the `Button`. `children` is more extendable where as if the button now can only accept a string, no emojis or icons or anything we can't think of today. We should also probably name the click event prop `onClick` so that it matches most of reacts API and is easily understandable.

I don't think a button component should be wrapped in a div either. That adds a lot of browser specific styles that are not associated with buttons normally.

```jsx
<Button onClick={onClick}>This is how buttons should work in general</Button>
<Button onClick={onClick}><Icon.Facebook/> Look, this button can take a facebook icon.</Button>
```

Lastly, this is newer but a good to do for all components that take user input. We should be passing the ref with the `forwardRef` api. So our button component becomes the example below.

```javascript
import React from "react"

// onClick has a default of an empty function.
export default React.forwardRef((({children, onClick = () => null}), ref) => {
    const styles = {
        // adding the inline block to the button itself gives a good mixture of a div and buttons reflow properties.
        display: "inline-block",
        height: "2em",
        fontSize: "1.5em",
        borderRadius: "3px",
        cursor: "pointer"
    }
    return <button style={styles} ref={ref} onClick={onClick}>{children}</button>
})
```

Reply:
Hey @pkrawc thanks for the helpful styling tips! Just for fun, I might work on incorporating these styling points locally and then could do a different PR, though I'm assuming we'll probably do much more styling than this to get the buttons looking better.